services/pkgreview/package-rating.service.js

```diff
@@ -74,26 +73,22 @@ module.exports = class PkgreviewRating extends BaseJsonService {
       schema,
       url: `https://pkgreview.dev/api/v1/${pkgManager}/${pkgSlug}`,
       errorMessages: {
-        400: 'package not found',
+        400: 'bad request',
+        404: 'package not found',
         408: 'response timed out',
-        502: 'response timed out',
-        500: 'response timed out',
       },
     })
   }
 
-  transform({ pkgSlug, ...json }) {
-    return {
-      ...json,
-      pkgSlug: encodeURIComponent(pkgSlug),
-    }
-  }
-
   async handle({ format, pkgManager, pkgSlug }) {
-    const json = await this.fetch({ pkgManager, pkgSlug })
-
-    const transformedJson = this.transform(json)
-
-    return this.constructor.render({ ...transformedJson, format })
+    const { reviewsCount, rating } = await this.fetch({
+      pkgManager,
+      pkgSlug: encodeURIComponent(pkgSlug),
+    })
+    return this.constructor.render({
+      reviewsCount,
+      format,
+      rating: +parseFloat(rating * 5).toFixed(1),
```

Comment:
I'd deliberately put `toFixed` within the `render` function in the code suggestions as that's really only necessary as part of the rendering when format is `rating` then actual rating value is included in the rendered badge message (because we wouldn't want to display a message like `4.458323413/5 (9k)` 😉)

Reply:
Will do!

Reply:
Fixed :)